### ISSUE TITLE
Everything is black&white after ugprade

### ISSUE THREAD

**Issue by roland-5** [neomutt/neomutt#4058]
## Expected Behaviour

Show colors.

## Actual Behaviour

Before open up, it show this message:
```
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 145: fabd2f: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 146: fabd2f: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 147: fabd2f: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 148: fabd2f: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 150: b8bb26: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 151: fb4934: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 152: b8bb26: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 153: fb4934: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 157: 83a598: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 158: 8ec07c: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 159: d3869b: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 160: fe8019: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 161: fabd2f: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 162: b8bb26: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 163: ebdbb2: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 164: fb4934: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 168: b8bb26: colour not supported by term
Error in /home/roland/.config/neomutt/gruvbox.muttrc, line 169: b8bb26: colour not supported by term
Error in /home/roland/.config/neomutt/neomuttrc, line 85: source: file /home/roland/.config/neomutt/gruvbox.muttrc could not be sourced
source: errors in /home/roland/.config/neomutt/neomuttrc
Press any key to continue...
```

and then open with everything black and white.

## Steps to Reproduce
My `neomuttrc`
```
# Default account
source "~/.config/neomutt/account1rc"

# Folder hooks
folder-hook 'account1rc' 'source ~/.config/neomutt/account1rc'
folder-hook 'account2rc' 'source ~/.config/neomutt/account2rc'

# Macros to switch accounts
macro index,pager <f2> '<sync-mailbox><enter-command>source ~/.config/neomutt/account1rc<enter><change-folder>!<enter>'
macro index,pager <f3> '<sync-mailbox><enter-command>source ~/.config/neomutt/account2rc<enter><change-folder>!<enter>'
macro index,pager <f4> '<sync-mailbox><enter-command>source ~/.config/neomutt/accounts/school<enter><change-folder>!<enter>'

# Mappings
## Fetch mail shortcut
bind index G imap-fetch-mail

set imap_keepalive = 900
set imap_check_subscribed
set ssl_force_tls = yes
set ssl_starttls = yes
bind index D purge-message
mailboxes = "+INBOX"
set move = no
set charset = "UTF-8"
set mail_check = 2
set timeout = 10
set date_format = "%y-%m-%d %T"
#set index_format="%4C %Z %{%m/%d} %-15.15F (%4c) %s" # format of the index
set index_format = "%2C | %Z [%d] %-30.30F (%-4.4c) %s"
# set index_format='%4C %Z %{%b %d} %-15.15L (%?l?%4l&%4c?) %s'
set mailcap_path = "~/.config/neomutt/mailcap"
set reply_to
# set status_format='-%r-NeoMutt: %f [Msgs:%?M?%M/?%m%?n? New:%n?%?o? Old:%o?%?d? Del:%d?%?F? \
# Flag:%F?%?t? Tag:%t?%?p? Post:%p?%?b? Inc:%b?%?l? %l?]---(%s/%S)-%>-(%P)---'
set use_threads=no sort=date sort_aux=date
set header_cache_compress_method = "zstd" #compression reduce size of header cache on disk, but increase ̣startup time
set header_cache_compress_level = 9
set sleep_time = 0
set pager_context = 3
set pager_stop = yes
set fast_reply = yes
set include = yes

# Set preffered view modes
auto_view text/html text/calendar application/ics # view html automatically
alternative_order text/html text/plain text/enriched text/*

set editor = "nvim"
#set editor = "$EDITOR"
#Attachments
macro attach E <save-entry><bol>~/attachments/<enter>
## Abook
#set query_command= "abook --mutt-query '%s'"
#macro index,pager a "<pipe-message>abook --add-email-quiet<return>" "Add this sender to Abook"
#bind editor        <Tab> complete-query

## MIME Support
set query_command = "notmuch-addrlookup %s"

## SIDEBAR
set sidebar_visible
set sidebar_width = 25
set sidebar_divider_char = '│'
set sidebar_format = "%B%?F? [%F]?%* %?N?%N/?%S"
set mail_check_stats
bind index,pager \CP sidebar-prev
bind index,pager \CN sidebar-next
bind index,pager \CO sidebar-open
bind index,pager B sidebar-toggle-visible
# Move the highlight to the previous page
# This is useful if you have a LOT of mailboxes.
bind index,pager <F3> sidebar-page-up
# Move the highlight to the next page
# This is useful if you have a LOT of mailboxes.
bind index,pager <F4> sidebar-page-down
# Move the highlight to the previous mailbox containing new, or flagged,
# mail.
bind index,pager <F5> sidebar-prev-new
# Move the highlight to the next mailbox containing new, or flagged, mail.
bind index,pager <F6> sidebar-next-new

## THEME
set color_directcolor = yes
source ~/.config/neomutt/gruvbox.muttrc
# vim: syntax=neomuttrc
```
My `gruvbox.muttrc`
```
## Base
color normal                    #ebdbb2 #000000 # good Default colour for all text

## Weak
color attachment                #d3869b #000000 # good Colour for attachment headers
color markers                   #000000 #000000 # good The "+" markers at the beginning of wrapped lines in the pager
color tilde                     #000000 #000000 # good The "~" used to pad blank lines in the pager
color tree                      #ebdbb2 #000000 # good Thread tree drawn in the message index and attachment menu
color signature                 #ebdbb2 #000000 # good Email's signature lines (.sig)

## Strong
# color bold                      #fbf1c7 #000000 # Highlighting bold patterns in the body of messages
# color underline                 #ebdbb2 #504945 # Highlighting underlined patterns in the body of messages

## Highlight
color error                     #fb4934 #000000 # good Error messages printed by NeoMutt
color indicator                 #ebdbb2 #3c3836 # good Arrow or bar used to indicate the current item in a menu
color message                   #000000 #b8bb26 # good Informational messages
color search                    #000000 #fe8019 # good Highlighting of words in the pager
color status                    #8ec07c #000000 # good Status bar

## Sidebar
color sidebar_divider           #000000 #000000 # good The dividing line between the Sidebar and the Index/Pager panels
color sidebar_flagged           #fabd2f #000000 # good Mailboxes containing flagged mail
# color sidebar_highlight color00  color167 #Cursor to select a mailbox
color sidebar_indicator         #ebdbb2 #3c3836 # good The mailbox open in the Index panel
color sidebar_new               #b8bb26 #000000 # good Mailboxes containing new mail
color sidebar_ordinary          #ebdbb2 #000000 # good Mailboxes that have no new/flagged mails, etc
# color sidebar_spool_file color00  color142 #Mailbox that receives incoming mail
color sidebar_unread            #fe8019 #000000 # good Mailboxes containing unread mail

# Message Index ----------------------------------------------------------------
## Weak
color index                     #83a598 #000000 "~R" # good read messages
# color index  color167  color214  "~v~(!~N)"  # #000000 collapsed thread with no unread
color index                     #d3869b #000000 "~Q" # good messages that have been replied to

## Strong
color index                     #fe8019 #000000 "~U" # good unread messages
# color index                     #b8bb26 #000000 ~N # The mail is New (Unread but not seen).
color index                     #000000 #fe8019 "~O" # good The mail is Old (Unread but seen).
# color index  color21  color00  "~v~(~U|~N|~O)"  # collapsed thread with unread

## Highlight
### Trusted
# color index  color06  color00  "~g"  # #000000 PGP signed messages
# color index  color06  color00  "~G"  # #000000 PGP encrypted messages
### Odd
# color index  color01  color00  "~E"  # #000000 past Expires: header date
# color index                     color167 color214  ~= #fb4934 duplicated
# color index  color01  color00  "~S"  # #000000 marked by Supersedes: header
### Flagged
color index                     #fabd2f #000000 "~F" # good flagged messages
# color index  color02  color00  "~v~(~F)"  # #000000 collapsed thread with flagged inside

## Selection
# color index                     color00  color248 ~T #000000 #bdae93 tagged messages
color index                     #000000 #fb4934 "~D" # good The mail is marked for deletion.
# color index                     #000000 #fb4934 "~d" # good The mail has attachments marked for deletion.


color index                     #fb4934 #000000 "~P" # good sent by me messages to me
# color index                     color00  color142 ~*  #000000      # The mail is tagged.
# color index                     color00  color142 ~! #000000       # The mail is flagged as important.
# color index                     color00  color142 ~r #000000       # The mail has been replied to.
# color index                     color00  color142 ~o #000000       # The mail thread is Old (Unread but seen).
# color index                     color00  color142 ~n #000000       # The mail thread is New (Unread but not seen).
# color index                     color00  color142 ~- #000000       # The mail is read - %S expando.
# color index                     color00  color142 ~<space> #000000 # The mail is read - %Z expando.

# Message Headers ----------------------------------------------------
## Base
color hdrdefault                #a89984 #000000 # good
## Strong
color header                    #a89984 #000000 "^Date:" # good
color header                    #83a598 #000000 "^From:" # good
color header                    #a89984 #000000 "^To:" # good
color header                    #a89984 #000000 "^CC:" # good
## Highlight
color header                    #b8bb26 #000000 "^Subject:" # good
color header                    #a89984 #000000 "^X-Mailer:" # good
# color header                    color108 color00 "^X-Spam-Status:" #8ec07c #000000
# color header                    color108 color167 "^Received:"     #8ec07c #000000

# Message Composer ----------------------------------------------------
## Base
color compose header            #ebdbb2 #000000 # good
## Security
color compose security_encrypt  #b8bb26 #000000 # good
color compose security_sign     #83a598 #000000 # good
color compose security_both     #d3869b #000000 # good
color compose security_none     #fb4934 #000000 # good

# Message Body -------------------------------------------------------

color body                      #fb4934 #000000 "[a-z]{3,255}://[[:graph:]]*" # good
color body                      #b8bb26 #000000 "([-[:alnum:]]+\\.)+([0-9]{1,3}|[-[:alpha:]]+)/[[:graph:]]*" # good
color body                      #b8bb26 #000000 "([-[:alnum:]]+\\.){2,255}[-[:alpha:]]{2,10}" # good

## Weak
### ~~~ Horizontal rules ~~~
# color body  color08  color00  "([[:space:]]*[-+=#*~_]){3,}[[:space:]]*" #000000

## Strong
### *Bold* span
# color body  color03  color00  "(^|[[:space:][:punct:]])\\*[^*]+\\*([[:space:][:punct:]]|$)" #000000

### _Underline_ span
# color body  color05  color00  "(^|[[:space:][:punct:]])_[^_]+_([[:space:][:punct:]]|$)" #000000
### /Italic/ span (Sometimes gets directory names)
# color body  color05  color00  "(^|[[:space:][:punct:]])/[^/]+/([[:space:][:punct:]]|$)" #000000
### ATX headers
# color body  color04  color00  "^[[:space:]]{0,3}#+[[:space:]].*$" #000000

## Highlight
### `Code` span
# color body  color02  color00  "(^|[[:space:][:punct:]])\`[^\`]+\`([[:space:][:punct:]]|$)" #000000
### Indented code block
# color body  color02  color00  "^[[:space:]]{4,}.*$" #000000
### URLs
# color body  color16 color00  "([a-z][a-z0-9+-]*://(((([a-z0-9_.!~*'();:&=+$,-]|%[0-9a-f][0-9a-f])*@)?((([a-z0-9]([a-z0-9-]*[a-z0-9])?)\\.)*([a-z]([a-z0-9-]*[a-z0-9])?)\\.?|[0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9]+)(:[0-9]+)?)|([a-z0-9_.!~*'()$,;:@&=+-]|%[0-9a-f][0-9a-f])+)(/([a-z0-9_.!~*'():@&=+$,-]|%[0-9a-f][0-9a-f])*(;([a-z0-9_.!~*'():@&=+$,-]|%[0-9a-f][0-9a-f])*)*(/([a-z0-9_.!~*'():@&=+$,-]|%[0-9a-f][0-9a-f])*(;([a-z0-9_.!~*'():@&=+$,-]|%[0-9a-f][0-9a-f])*)*)*)?(\\?([a-z0-9_.!~*'();/?:@&=+$,-]|%[0-9a-f][0-9a-f])*)?(#([a-z0-9_.!~*'();/?:@&=+$,-]|%[0-9a-f][0-9a-f])*)?|(www|ftp)\\.(([a-z0-9]([a-z0-9-]*[a-z0-9])?)\\.)*([a-z]([a-z0-9-]*[a-z0-9])?)\\.?(:[0-9]+)?(/([-a-z0-9_.!~*'():@&=+$,]|%[0-9a-f][0-9a-f])*(;([-a-z0-9_.!~*'():@&=+$,]|%[0-9a-f][0-9a-f])*)*(/([-a-z0-9_.!~*'():@&=+$,]|%[0-9a-f][0-9a-f])*(;([-a-z0-9_.!~*'():@&=+$,]|%[0-9a-f][0-9a-f])*)*)*)?(\\?([-a-z0-9_.!~*'();/?:@&=+$,]|%[0-9a-f][0-9a-f])*)?(#([-a-z0-9_.!~*'();/?:@&=+$,]|%[0-9a-f][0-9a-f])*)?)[^].,:;!)? \t\r\n<>\"]" #000000
# Mail addresses and mailto URLs
# color body  color16 color00  "((@(([0-9a-z-]+\\.)*[0-9a-z-]+\\.?|#[0-9]+|\\[[0-9]?[0-9]?[0-9]\\.[0-9]?[0-9]?[0-9]\\.[0-9]?[0-9]?[0-9]\\.[0-9]?[0-9]?[0-9]\\]),)*@(([0-9a-z-]+\\.)*[0-9a-z-]+\\.?|#[0-9]+|\\[[0-9]?[0-9]?[0-9]\\.[0-9]?[0-9]?[0-9]\\.[0-9]?[0-9]?[0-9]\\.[0-9]?[0-9]?[0-9]\\]):)?[0-9a-z_.+%$-]+@(([0-9a-z-]+\\.)*[0-9a-z-]+\\.?|#[0-9]+|\\[[0-2]?[0-9]?[0-9]\\.[0-2]?[0-9]?[0-9]\\.[0-2]?[0-9]?[0-9]\\.[0-2]?[0-9]?[0-9]\\])"
color body                      #fe8019 #000000 "[-a-z_0-9.%$]+@[-a-z_0-9.]+\\.[-a-z][-a-z]+"
color body                      #fe8019 #000000 "mailto:[-a-z_0-9.]+@[-a-z_0-9.]+"

## Emoticons simleys and stuff
# color body                       black   yellow   "[;:][-o]?[})>{(<|P]"
color body                      #000000 #fabd2f "[;:]-*[)>(<lt;|]"
color body                      #fbf1c7 #000000 "\\*[- A-Za-z]+\\*"

## PGP
color body  color21  color01   "(BAD signature)"
color body  color21  color01   "^gpg: BAD signature from.*"
color body  color21  color04   "(Good signature)"
color body  color21  color04   "^gpg: Good signature .*"
color body  color04  color00   "^gpg: "

color body                      #fabd2f #000000 "^-.*PGP.*-*"
color body                      #b8bb26 #000000 "^gpg: Good signature from"
color body                      #fb4934 #000000 "^gpg: Can't.*$"
color body                      #fabd2f #000000 "^gpg: WARNING:.*$"
color body                      #fb4934 #000000 "^gpg: BAD signature from"
color body                      #fb4934 #000000 "^gpg: Note: This key has expired!"
color body                      #fabd2f #000000 "^gpg: There is no indication that the signature belongs to the owner."
color body                      #fabd2f #000000 "^gpg: can't handle these multiple signatures"
color body                      #fabd2f #000000 "^gpg: signature verification suppressed"
color body                      #fabd2f #000000 "^gpg: invalid node with packet of type"

color body                      #b8bb26 #000000 "^Good signature from:"
color body                      #fb4934 #000000 "^.?BAD.? signature from:"
color body                      #b8bb26 #000000 "^Verification successful"
color body                      #fb4934 #000000 "^Verification [^s][^[:space:]]*$"

# Quotation blocks ---------------------------------------------------
##Base
color quoted                    #83a598 #000000 # good
color quoted1                   #8ec07c #000000 # good
color quoted2                   #d3869b #000000 # good
color quoted3                   #fe8019 #000000 # good
color quoted4                   #fabd2f #000000 # good
color quoted5                   #b8bb26 #000000 # good
color quoted6                   #ebdbb2 #000000 # good
color quoted7                   #fb4934 #000000 # good


# IPv4 and IPv6 stolen from https://stackoverflow.com/questions/53497/regular-expression-that-matches-valid-ipv6-addresses
color body                      #b8bb26 #000000 "((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])"
color body                      #b8bb26 #000000 "(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))"

# vim: syntax=neomuttrc
```

## How often does this happen?

Always

## When did it start to happen?

- When I upgraded
I used [2023-05-17](https://github.com/neomutt/neomutt/releases/tag/20230517) and now [2023-10-06](https://github.com/neomutt/neomutt/releases/tag/20231006).

## NeoMutt Version

```
NeoMutt 20231006
Copyright (C) 1996-2022 Michael R. Elkins and others.
NeoMutt comes with ABSOLUTELY NO WARRANTY; for details type 'neomutt -vv'.
NeoMutt is free software, and you are welcome to redistribute it
under certain conditions; type 'neomutt -vv' for details.

System: Linux 6.5.6-arch1-1 (x86_64)
ncurses: ncurses 6.4.20230520 (compiled with 6.4.20230520)
libidn2: 2.3.4 (compiled with 2.3.4)
GPGME: 1.22.0
GnuTLS: 3.8.1
libnotmuch: 5.6.0
storage: kyotocabinet, gdbm, lmdb
compression: lz4, zlib, zstd

Configure options: --prefix=/usr --sysconfdir=/etc --libexecdir=/usr/lib --gpgme --sqlite --autocrypt --lua --notmuch --gss --gnutls --sasl --with-ui=ncurses --with-idn2=/usr --disable-idn --idn2 --lmdb --kyotocabinet --gdbm --ubsan --lz4 --zlib --zstd

Compilation CFLAGS: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -g -ffile-prefix-map=/build/neomutt/src=/usr/src/debug/neomutt -flto=auto -fno-delete-null-pointer-checks -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -D_XOPEN_SOURCE_EXTENDED -I/usr/include/lua5.3 -I/usr/include -I/usr/include -I/usr/include -DNCURSES_WIDECHAR -I/usr/include/p11-kit-1 -I/usr/include -I/usr/include -fsanitize=undefined -fno-omit-frame-pointer -fno-common -g -O0

Default options:
  +attach_headers_color +compose_to_sender +compress +cond_date +debug
  +encrypt_to_self +forgotten_attachments +forwref +ifdef +imap +index_color
  +initials +limit_current_thread +multiple_fcc +nested_if +new_mail +nntp +pop
  +progress +quasi_delete +regcomp +reply_with_xorig +sensible_browser +sidebar
  +skip_quoted +smtp +status_color +timeout +tls_sni +trash

Compile options:
  +autocrypt +fcntl -flock -fmemopen +futimens +getaddrinfo +gnutls +gpgme
  -gsasl +gss +hcache -homespool +idn +inotify -locales_hack +lua -mixmaster
  +nls +notmuch -openssl +pgp +regex +sasl +smime +sqlite +sun_attachment

Devel options:
  ubsan

MAILPATH="/var/mail"
PKGDATADIR="/usr/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/etc"

To learn more about NeoMutt, visit: https://neomutt.org
If you find a bug in NeoMutt, please raise an issue at:
    https://github.com/neomutt/neomutt/issues
or send an email to: <neomutt-devel@neomutt.org>
```

## Extra Info
* Operating System and its version
Arch Linux
* Were you using multiple copies of NeoMutt at once?
No
* Were you using 'screen' or 'tmux'?
No
* Is your email local (maildir) or remote (IMAP)?
IMAP

**Comment by flatcap** [comment]
This is a bit odd.
Your `gruvbox.muttrc` looks OK to me in both **2023-05-17** and **2023-10-06**.

You've forcibly `set color_directcolor = yes` so NeoMutt ought to accept the config.
Unless, ncurses thinks the terminal can't handle that many colours.

Can you check your environment variables:
You need **both**:
```sh
export COLORTERM=truecolor
export TERM=xterm-direct
```

If they're correct, try a simple config:

**test.rc**
```sh
color status red      green
color status color123 color234
color status #112233  #223344
set color_directcolor = yes
color status #556677  #667788
```

Then run NeoMutt with **just** this config, and let me know all the errors that you see
```sh
neomutt -n -F test.rc
```

Thanks

**Comment by roland-5** [comment]
For alacritty:
```
echo $COLORTERM
truecolor
```
```
echo $TERM
alacritty
```
For wezterm
```
echo $COLORTERM
truecolor
```
```
echo $TERM
xterm-256color
```

For both alacritty and wezterm:
```
neomutt -n -F test.rc
Error in /home/roland/.config/neomutt/test.rc, line 3: Direct colors support disabled: #112233
Error in /home/roland/.config/neomutt/test.rc, line 5: 556677: colour not supported by term
source: errors in /home/roland/.config/neomutt/test.rc
```

![ps_20231007125632](https://github.com/neomutt/neomutt/assets/44477799/78be0d83-0171-4589-912d-86b8ef263f24)

**Comment by flatcap** [comment]
OK, you need to: `export TERM=xterm-direct` to make sure that ncurses knows the terminal's capabilities.
Try that, then see if the test works.

**Comment by roland-5** [comment]
That helped. Thank you!

**Comment by flatcap** [comment]
Great! :-)

**Comment by tkapias** [comment]
Just to comment that I had almost the same issue in the same period, but with a different cause.

My previous build ([186c63f](https://github.com/neomutt/neomutt/commit/186c63f8f8ece4c9c38d2b3b9270323d4205cb8f)) was working fine with true colors inside Tmux. I used an alias (`TERM=tmux-direct neomutt -f "📫Boite à lettres"'`) and forced `set color_directcolor = yes`.

But, when updating to a new build this morning I got the same errors as @roland-5.

After reading this thread the only difference seemed to be that my color values were double-quoted (it worked), so I tried to unquote and it solved the issue with the current build.

It may help someone else.
